src/scss/components/optional/_order-card.scss

```diff
@@ -103,6 +103,10 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
         @include font-size(base--scaleUp, false);
         font-weight: $font-weight-bold;
         line-height: line-height(16px, 24px);
+        font-family: $font-family-headings;
+        font-weight: $font-weight-base;
+        @include font-size(22px);
+        padding-top: 2px;
```

Comment:
Can you add this style to the `padding` rule below. [It's causing a lint error](https://travis-ci.org/justeat/fozzie/jobs/605972644#L244).